src/Makefile.mk

```diff
@@ -5,6 +5,9 @@ KUBEAPPLY ?= kubectl-apply
 HELM_TMP_GENERATE ?= /tmp/generate
 HELM_TMP_SECRETS ?= /tmp/secrets/jx-helm
 
+# Prevent accidental git commit of secrets
+HELM_TMP_SECRETS_WORKSPACE ?= /workspace/source/tmp/secrets/jx-helm
```

Comment:
The only reason not to do `HELM_TMP_SECRETS_WORKSPACE ?= /workspace/source/$(HELM_TMP_SECRETS)` is that if someone sets `HELM_TMP_SECRETS` to `/`, then this can wipe out the git repo.